tests/input/fhir_npm_manifest_so.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
@cmungall I included a static manifest JSON. Do you prefer I instead declare this as a dictionary instead as you have [set up here](https://github.com/INCATools/ontology-access-kit/blob/main/tests/test_cli.py#L453-L457)?

Reply:
When linking to code blocks I recommend doing this for a specific commit hash as this no longer resolves to the lines it once did

Reply:
That's a good idea. Here's [the original snippet](https://github.com/INCATools/ontology-access-kit/blob/f7d37380f5b92b8718176915298935dd03cfc67f/tests/test_cli.py#L503-L507).

Reply:
Marking resolved as now duplicative with this comment:
- https://github.com/INCATools/ontology-access-kit/pull/511#discussion_r1159231846